scenarios/ocd/CreateStaticWebApp/README.md

```diff
@@ -0,0 +1,143 @@
+# Azure Static Web Apps Quickstart: Building Your First Static Site Using the Azure CLI
+
+Azure Static Web Apps publishes websites to production by building apps from a code repository. In this quickstart, you deploy a web application to Azure Static Web Apps using the Azure CLI.
+
+## Define Environment Variables
+
+The First step in this tutorial is to define environment variables.
+
+```bash
+export RANDOM_ID="$(openssl rand -hex 3)"
+export MY_RESOURCE_GROUP_NAME="myStaticWebAppResourceGroup$RANDOM_ID"
+export REGION=EastUS2
+export MY_STATIC_WEB_APP_NAME="myStaticWebApp$RANDOM_ID"
+```
+
+## Create a Repository (optional)
+
+(Optional) This article uses a GitHub template repository as another way to make it easy for you to get started. The template features a starter app to deploy to Azure Static Web Apps.
+
+- Navigate to the following location to create a new repository: https://github.com/staticwebdev/vanilla-basic/generate
+- Name your repository `my-first-static-web-app`
+
+> **Note:** Azure Static Web Apps requires at least one HTML file to create a web app. The repository you create in this step includes a single `index.html` file.
+
+Select `Create repository`.
+
+## Deploy a Static Web App
+
+You can deploy the app as a static web app from the Azure CLI.
+
+1. Create a resource group.
+
+   ```bash
+   az group create \
+     --name $MY_RESOURCE_GROUP_NAME \
+     --location $REGION
+   ```
+
+Results:
+
+<!-- expected_similarity=0.3 -->
+```json
+{
+  "id": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/my-swa-group",
+  "location": "eastus2",
+  "managedBy": null,
+  "name": "my-swa-group",
+  "properties": {
+    "provisioningState": "Succeeded"
+  },
+  "tags": null,
+  "type": "Microsoft.Resources/resourceGroups"
+}
+```
+
+2. Deploy a new static web app from your repository.
+
+   ```bash
+   az staticwebapp create \
+       --name $MY_STATIC_WEB_APP_NAME \
+       --resource-group $MY_RESOURCE_GROUP_NAME \
+       --location $REGION 
+   ```
+
+There are two aspects to deploying a static app. The first operation creates the underlying Azure resources that make up your app. The second is a workflow that builds and publishes your application.
+
+Before you can go to your new static site, the deployment build must first finish running.
+
+3. Return to your console window and run the following command to list the website's URL.
+
+```bash
+export MY_STATIC_WEB_APP_URL=$(az staticwebapp show --name  $MY_STATIC_WEB_APP_NAME --resource-group $MY_RESOURCE_GROUP_NAME --query "defaultHostname" -o tsv)
+```
+
+```bash
+runtime="1 minute";
+endtime=$(date -ud "$runtime" +%s);
+while [[ $(date -u +%s) -le $endtime ]]; do
+    if curl -I -s $MY_STATIC_WEB_APP_URL > /dev/null ; then 
+        curl -L -s $MY_STATIC_WEB_APP_URL 2> /dev/null | head -n 9
```

Comment:
How come we're only comparing the first 9 lines of the web page? Is that expected?